thumbnails/CHANGELOG.md

```diff
@@ -0,0 +1,147 @@
+* Change - Refactor code to remove code smells: [#21](https://github.com/owncloud/ocis/thumbnails/issues/21)
+* Change - Use micro service error api: [#31](https://github.com/owncloud/ocis/thumbnails/issues/31)
+* Enhancement - Limit users to access own thumbnails: [#5](https://github.com/owncloud/ocis/thumbnails/issues/5)
+
+## Details
+
+* Change - Add more default resolutions: [#23](https://github.com/owncloud/ocis/thumbnails/issues/23)
+
+   The `ocis-thumbnails` service was also used by the mediaviewer but the returned images were
+   too small since the highest resolution was 128x128 pixels.
+
+   https://github.com/owncloud/ocis/thumbnails/issues/23
+
+
+* Change - Refactor code to remove code smells: [#21](https://github.com/owncloud/ocis/thumbnails/issues/21)
+
+   Scanning the code using a static code analyzer showed some code smells. This change fixes them.
+
+   https://github.com/owncloud/ocis/thumbnails/issues/21
+
+
+* Change - Use micro service error api: [#31](https://github.com/owncloud/ocis/thumbnails/issues/31)
+
+   The service now also returns a status code when an error occurs.
+
+   https://github.com/owncloud/ocis/thumbnails/issues/31
+
+
+* Enhancement - Limit users to access own thumbnails: [#5](https://github.com/owncloud/ocis/thumbnails/issues/5)
+
+   Users of the service can no longer request thumbnails of another users by guessing the etag. The
+   thumbnails are now only accessible by the users who created the thumbnail.
+
+   https://github.com/owncloud/ocis/thumbnails/issues/5
+
+# Changelog for [0.1.2] (2020-05-11)
+
+The following sections list the changes in ocis-thumbnails 0.1.2.
+
+[0.1.2]: https://github.com/owncloud/ocis/thumbnails/compare/v0.1.1...v0.1.2
+
+## Summary
+
+* Bugfix - Fix usage of context.Context: [#18](https://github.com/owncloud/ocis/thumbnails/issues/18)
+
+## Details
+
+* Bugfix - Fix usage of context.Context: [#18](https://github.com/owncloud/ocis/thumbnails/issues/18)
+
+   The context was filled with a key defined in the package service but read with a key from the
+   package imgsource. Since `service.key` and `imgsource.key` are different types imgsource
+   could not read the value provided by service.
+
+   https://github.com/owncloud/ocis/thumbnails/issues/18
+
+# Changelog for [0.1.1] (2020-04-21)
+
+The following sections list the changes in ocis-thumbnails 0.1.1.
+
+[0.1.1]: https://github.com/owncloud/ocis/thumbnails/compare/v0.1.0...v0.1.1
+
+## Summary
+
+* Bugfix - Fix execution when passing program flags: [#15](https://github.com/owncloud/ocis/thumbnails/issues/15)
+
+## Details
+
+* Bugfix - Fix execution when passing program flags: [#15](https://github.com/owncloud/ocis/thumbnails/issues/15)
+
+   The program flags where not correctly recognized because we didn't pass them to the micro
+   framework when initializing a grpc service.
+
+   https://github.com/owncloud/ocis/thumbnails/issues/15
+
+# Changelog for [0.1.0] (2020-03-31)
```

Comment:
Codacy found an issue: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6202733)